experiments/ssl4eo/compute_dataset_statistics.py

```diff
@@ -40,11 +40,11 @@ def compute(path: str) -> tuple["np.typing.NDArray[np.float32]", int]:
     )
 
     if args.num_workers > 0:
-        out_list, size_list = list(
+        out_tuple, size_tuple = list(
```

Comment:
As far as I can tell, this is the only real "bug" uncovered by mypy. `thread_map` returns tuples, not lists. In the else-statement, we are enforcing that these are lists.